modules/openapi-generator/src/main/resources/cpp-qt5-client/api-body.mustache

```diff
@@ -11,9 +11,9 @@ namespace {{this}} {
 
 {{classname}}::{{classname}}(const QString &scheme, const QString &host, int port, const QString& basePath, const int timeOut) :
     _scheme(scheme),
+    _basePath(basePath),
     _host(host),
     _port(port),
-    _basePath(basePath),
```

Comment:
Sorry why are we changing the order here given that _basePath is the second last parameter in line 12?

Reply:
Ok I rather changed the private member declaration order to be consistent.